app/controllers/assessments_controller.rb

```diff
@@ -21,10 +19,10 @@ def index
     assessments = patient&.assessments
     redirect_to(root_url) && return if patient.nil? || assessments.nil?
 
-    assessments = search(assessments, search_text)
-    assessments = sort(assessments, sort_order, sort_direction)
-    assessments = paginate(assessments, entries, page)
-    assessments = format_for_frontend(assessments)
+    assessments = AssessmentQueryHelper.search(assessments, search_text)
+    assessments = AssessmentQueryHelper.sort(assessments, sort_order, sort_direction)
+    assessments = AssessmentQueryHelper.paginate(assessments, entries, page)
+    assessments = AssessmentQueryHelper.format_for_frontend(assessments)
```

Comment:
Why don't we have a test file for `AssessmentQueryHelper`'s methods?

Reply:
seems like tests were just never written for the assessment query helper yet. I would offer to do it, but I think that might be outside the scope of this ticket (other than the `format_for_frontend` method since this is more focused on the assessment table loading performance). I have included system tests in this PR that verify each assessment, its symptom values, the symptomatic styling/color of each cell and row, etc, which I think is sufficient enough for now to verify the correctness of the code being changed